core/src/main/java/com/clubobsidian/dynamicgui/function/impl/condition/AfterDateLazyFunction.java

```diff
@@ -0,0 +1,32 @@
+package com.clubobsidian.dynamicgui.function.impl.condition;
+
+import com.udojava.evalex.AbstractLazyFunction;
+import com.udojava.evalex.Expression.LazyNumber;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.time.Instant;
+import java.util.Date;
+import java.util.List;
+
+public class AfterDateLazyFunction extends AbstractLazyFunction {
+
+    protected AfterDateLazyFunction() {
+        super("AFTRDATE", 1, true);
+    }
+
+    @Override
+    public LazyNumber lazyEval(List<LazyNumber> lazyParams) {
+        try {
+            Date now = Date.from(Instant.now());
+            Date expected = new SimpleDateFormat("MM dd, yyyy HH:mm:ss").parse(lazyParams.get(0).getString());

```

Comment:
The date time format should probably be configurable, config values can be loaded here.
https://github.com/ClubObsidian/DynamicGui/blob/master/core/src/main/java/com/clubobsidian/dynamicgui/DynamicGui.java#L192